libs/data-types/README.md

```diff
@@ -0,0 +1,11 @@
+# data-types
```

Comment:
We could put a little description in here if needed

Reply:
I'll do this as a separate commit.